content/en/docs/concepts/services-networking/service.md

```diff
@@ -166,6 +170,10 @@ By default, the choice of backend is round robin.
 
 Note that in the above diagram, `clusterIP` is shown as `ServiceIP`.
 
+{{< note >}}
+**Note:** SCTP is not supported in userspace mode.
```

Comment:
How do we say this when we had that huge discussion about it?

Reply:
Yeah, here I mean the kube-proxy userspace mode. That is, if the kube-proxy is in userspace mode it does not proxy SCTP traffic through itself. But of course, the usage of userspace SCTP stacks by applications is allowed, as we discussed during the PR.

Reply:
Actually, I delete this note from here. There is a separate note about it in the main SCTP chapter at the bottom of this page.